src/prefect/client/orion.py

```diff
@@ -815,19 +817,24 @@ async def read_work_queues(
         body = {
             "limit": limit,
             "offset": offset,
+            "work_queues": (
+                work_queue_filter.dict(json_compatible=True)
+                if work_queue_filter
+                else None
+            ),
         }
         response = await self._client.post(f"/work_queues/filter", json=body)
         return pydantic.parse_obj_as(List[schemas.core.WorkQueue], response.json())
 
     async def match_work_queues(
         self,
-        prefixes: List[str],
```

Comment:
The feature that relies on this agent method may pass in a list of prefixes to match, could we keep that?

Reply:
It'll take some rework, but yeah. The tests are all setup to just send through a string (no list) so I _assumed_ the type here was just wrong.